File: csv/octave/update/csv_update.m
#! /usr/bin/octave -qf
#
#	csv_update.m
#
#					Nov/11/2014
#
# ------------------------------------------------------------------
path (path, "/var/www/data_base/common/octave_common");
# ------------------------------------------------------------------
printf ("*** 開始 ***\n")
arg_list = argv ();
file_txt = arg_list{1};
key_in = arg_list{2};
population_in = arg_list{3};
disp (file_txt)
disp (key_in)
disp (population_in)
#
dict_aa = struct;
dict_aa = csv_read_proc (file_txt)
#
if  (isfield (dict_aa, key_in))
	unit_aa = dict_aa.(key_in)
	unit_aa = setfield (unit_aa, "population", population_in)
	unit_aa = setfield (unit_aa, "date_mod", date ())
	dict_aa = setfield (dict_aa, key_in, unit_aa)
	dict_display_proc (dict_aa)
	csv_write_proc (file_txt,dict_aa)
endif
#
printf ("*** 終了 ***\n")
# ------------------------------------------------------------------

File: csv/nashorn/update/csv_update.js
#! /usr/bin/env jjs
// ------------------------------------------------------------------
//
//	csv_update.js
//
//						Jan/04/2017
//
// ------------------------------------------------------------------
load ("/var/www/data_base/common/nashorn_common/file_io.js")
load ("/var/www/data_base/common/nashorn_common/text_manipulate.js")

var file_name=arguments[0]
var key_in=arguments[1]
var population_in=arguments[2]

print	("*** 開始 ***")
print	("key_in = " + key_in + "\tpopulation_in = " + population_in)

var dict_in = csv_read_proc (file_name)
var dict_bb = dict_update_proc (dict_in,key_in,population_in)
csv_write_proc (file_name,dict_bb)

print	("*** 終了 ***")
// ------------------------------------------------------------------
